demos/showcase/router/navigation-drawer.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+    Drawer,
+    DrawerBody,
+    DrawerHeader,
+    DrawerNavGroup,
+    NavItem,
+    Subtitle,
+    wrapIcon,
+    DrawerFooter
+} from "@pxblue/react-native-components";
+import React, { useState, useCallback } from 'react';
+import MaterialCommunityIcon from "react-native-vector-icons/MaterialCommunityIcons";
+import MatIcon from "react-native-vector-icons/MaterialIcons";
+// @ts-ignore
+import _Humidity from '@pxblue/icons-svg/moisture.svg';
+// @ts-ignore
+import _Battery from '@pxblue/icons-svg/battery.svg';
+import {Image, View} from "react-native";
+import {Divider} from "react-native-elements";
+import {IconButton} from "react-native-paper";
+import * as Colors from '@pxblue/colors';
+
+const headerBgImage = require('../assets/images/topology_40.png');
+const eatonLogo = require('../assets/images/eatonLogo.png');
+const Battery = wrapIcon({ IconClass: _Battery });
+const Humidity = wrapIcon({ IconClass: _Humidity });
+const Clock = wrapIcon({ IconClass: MaterialCommunityIcon, name: 'clock-outline' });
+const MailIcon = wrapIcon({ IconClass: MatIcon, name: 'mail' });
+
+export const navGroupItems1: NavItem[] = [
+    {
+        title: 'Identity Management',
+        itemID: 'g1i1',
+        icon: Battery
+    },
+    {
+        title: 'Calendar',
+        itemID: 'g1i2',
+        icon: Humidity,
+    },
+    {
+        title: 'Accessibility',
+        itemID: 'g1i3',
+        icon: Clock,
+        subtitle: 'Sample subtitle',
+        onItemSelect: (): void => { /* Expand and don't update selected */ },
```

Comment:
This isn't here because it's required,  I put this empty function here because I put an inheritable `onItemSelect` function on the Drawer that sets the selected item on click.  Since itemID `g1i3` is an expandable item, I didn't want it to be marked as selected, so I put the empty function there to break the void the inheritance from the Drawer.

Reply:
I realize this is a little confusing so I'm going to push out an alternative solution.  Basically the new idea is that the `onItemSelect` function on the drawer will update the state, only if the itemID provided isn't an expandable NavItem.